src/Couchbase.Lite.Shared/API/Document/Blob.cs

```diff
@@ -280,12 +281,9 @@ internal void FLEncode(FLEncoder* enc)
                     WriteLog.To.Database.W(Tag, "Error installing blob to database, throwing...");
                     throw;
                 }
-
-                var dict = JsonRepresentation;
-                dict.FLEncode(enc);
+                JsonRepresentation.FLEncode(enc);
```

Comment:
Since this is used directly in both cases, just move it outside the `if`

Reply:
Right, duh!